projects/email-editor/src/types/tools.ts

```diff
@@ -0,0 +1,7885 @@
+// import Mustache from 'mustache';
+import { DisplayMode, DesignTags, Collection, Location, Device, Variant, DesignMode, Validator, DisplayConditions, Container } from './StateTypes';
```

Comment:
Do we really want a 7000 line file around? Does adding all the new types have any inpact on the end user node module package size? We can check with webpack bundle analizer

Reply:
Thanks for pointing out, i have reduced the Type definitions and kept only the ones required, i have also changed from using .ts files to d.ts only so we don't end up adding the .ts to our package.

The unpacked size is 150kb, (https://www.npmjs.com/package/@omerzahidbajwa/angular-email-editor), previously it was 70kb but `unlayer-types` dependency was another 100kb

I also had initially built out the type definition files form our editor/embed and then ported them over manually, i realize we are updating the types package on a regular basis and will end up loosing sync, but atleast this will give a good auto complete support for now till we can look into the `unlayer-types` package